flink-connectors/flink-connector-kafka-0.8/src/main/java/org/apache/flink/streaming/connectors/kafka/Kafka08JsonTableSink.java

```diff
@@ -42,7 +43,7 @@ public Kafka08JsonTableSink(String topic, Properties properties, FlinkKafkaParti
 	}
 
 	/**
-	 * Creates {@link KafkaTableSink} for Kafka 0.8
+	 * Creates {@link KafkaTableSink} for Kafka 0.8 .
```

Comment:
This is somewhat intended, to clarify that it is the end of the sentence and not an incomplete version. Don't mind removing the space though.